tests/format/angular/let-declaration/complex.html

```diff
@@ -0,0 +1,39 @@
+@let foo = a | b : c:d :e;
```

Comment:
Since the tests silently fails, please make the code uglier, I usually put 3 spaces in code.

And we parse the declarations one by one, so we have to make all of they uglier..

Reply:
https://github.com/prettier/prettier/pull/16474/commits/3b4b58f117bf73135e9b6090a0143fb535f919ef